### PR TITLE
Fixes for the Post Hackathon TownSquare Deploy - Needs Review

### PR DESCRIPTION
**Pull request by androolloyd**
##### Description

Post Deploy Fixes/Tweaks
- cleanup, post deploy fixes, and some layout fixes
- registration button
- add to channel bug fix
- ensures the old onboard url will still work

### CODE REVIEW

File: app/dashboard/templates/dashboard/index-vue.html [hunk 1]
```diff
@@ -482,8 +409,8 @@ <h3>{% trans "No results found." %}</h3>
                                 <div class="card-body">
                                 <h5 class="card-title font-weight-bold text-left">[[ project.name ]]</h5>
                                 <div class="my-2">
-                                  <p class="text-left text-muted font-smaller-1 summary">
-                                    [[ project.summary ]]
+                                  <p class="text-left text-muted font-smaller-1">
+                                    [[ project.summary | truncate(500) ]]
```
Comment (danlipert) [inline app/dashboard/templates/dashboard/index-vue.html:413]: nice!

File: app/assets/v2/js/vue-filters.js [hunk 2]
```diff
@@ -90,3 +90,26 @@ Vue.filter('stringReplace', function(activity_type) {
...
+Vue.filter('truncate', function(text, length, clamp) {
+  text = text || '';
+  clamp = clamp || '...';
+  length = length || 30;
+
+  if (text.length <= length)
+    return text;
+
+  var tcText = text.slice(0, length - clamp.length);
+  var last = tcText.length - 1;
```
Comment (thelostone-mc) [inline app/assets/v2/js/vue-filters.js:102]: ^ could we update those to const and let ?

File: app/assets/v2/js/vue-filters.js [hunk 3]
```diff
@@ -90,3 +90,25 @@ Vue.filter('stringReplace', function(activity_type) {
   return activity_names[activity_type];
 });
 
+Vue.filter('truncate', function(text, length, clamp) {
```
Comment (thelostone-mc) [inline app/assets/v2/js/vue-filters.js:93]: I've got a version for truncate -> when we rebase this against master
we'll have pick one of those 

https://github.com/gitcoinco/web/pull/6419

cc @octavioamu

Comment (androolloyd) [reply app/assets/v2/js/vue-filters.js]: using yours.

**Pull request closed without merging**